### ISSUE TITLE
Python Script Error

### ISSUE THREAD

**Issue by crnvl** [kiwi515/gen4-RPC#2]
I'm getting this Error every time I execute the Python Script.
´´´
from pypresence import Presence, Activity
ImportError: cannot import name 'Activity' from 'pypresence' (path)
´´´
The Reason for this might be a newer version of Pypresence, which one is recommended to use?

**Comment by kiwi515** [comment]
Hello! I think this issue is caused by the Activity class being deprecated in some previous release of pypresence. I'm currently looking into this and will hopefully release an update of this tool shortly, so that users can update to the newest release of pypresence without problems.

**Comment by kiwi515** [comment]
See if the [latest release](https://github.com/kiwi515/Gen4RPC/releases/tag/0.2.0) works for you now:

**Comment by crnvl** [comment]
```Traceback (most recent call last):
  File "rpc.py", line 65, in <module>
    if luaStatsList[gameNameOffset] != 4:
IndexError: list index out of range```

**Comment by kiwi515** [comment]
Ah, that occurs when out.dat exists but is empty. For the time being, run the Lua script until you see the Lua console output that you've entered a new map. At that point, out.dat will have enough data for the Python script to not check out of bounds.

I plan to soon refactor the design of this considering it is very old, but for the time being I'd recommend using the fix described above.

**Comment by kiwi515** [comment]
The issues described above have been corrected in release 0.3.0. Closing

**Comment by crnvl** [comment]
Okay thanks for fixing. Sadly i wanted to use it with Pkmn SS. let me know if you implement this in the near future!
